protos/google/cloud/dataproc/v1/clusters.proto

```diff
@@ -18,7 +18,7 @@ syntax = "proto3";
 package google.cloud.dataproc.v1;
 
 import "google/api/annotations.proto";
-import "google/cloud/dataproc/v1/operations.proto";
```

Comment:
@alexander-fenster I'm not sure if the removal of this import is intentional or not, but it looks like the gapics still depend on it and removing it causes our unit tests to fail.

Reply:
FYI @JustinBeckwith @sduskis

Reply:
The dataproc team updated this on their end: https://github.com/googleapis/googleapis/commit/8f6588f68e1f3fc2f6cbb47d12565f9b81926d42.  I'm going to close this PR, and hopefully the next autosynth will work better.